src/rgw/rgw_rest.cc

```diff
@@ -1572,6 +1572,12 @@ int RGWCompleteMultipart_ObjStore::get_params(optional_yield y)
   std::tie(op_ret, data) = read_all_input(s, max_size);
   if (op_ret < 0)
     return op_ret;
+  
+  const char *bypass_gov_header = s->info.env->get("HTTP_X_AMZ_BYPASS_GOVERNANCE_RETENTION");
+  if (bypass_gov_header) {
+    std::string bypass_gov_decoded = url_decode(bypass_gov_header);
+    bypass_governance_mode = boost::algorithm::iequals(bypass_gov_decoded, "true");
```

Comment:
`x-amz-bypass-governance-retention` isn't listed as a request parameter for https://docs.aws.amazon.com/AmazonS3/latest/API/API_CompleteMultipartUpload.html#API_CompleteMultipartUpload_RequestSyntax

can you help me understand why this is needed? i see new logic in `RGWCompleteMultipart::verify_permission()` to determine `bypass_perm` and update `bypass_governance_mode`, but i don't see `bypass_governance_mode` actually used anywhere

Reply:
i plan to push an additional commit to remove these changes from `RGWCompleteMultipart`, since we don't need to call `verify_object_lock()` here